src/interfaces.ts

```diff
@@ -0,0 +1,41 @@
+import {Constants} from './actions/types';
+
+export interface IProps {
```

Comment:
Props for each component should be in corresponding RN component
No need to use prefix "I" its a bad practice
So lets remove this file and put Props for each component separately, and define what we receive there (don't use any)